src/entry/logEntry.ts

```diff
@@ -13,6 +13,10 @@ export type LogEntry = {
   content: string;
 };
 
+export type IP = {
```

Comment:
To be correct it should be:
```suggestion
export type Ip = {
```